packages/fx-core/src/component/driver/aad/update.ts

```diff
@@ -100,6 +101,19 @@ export class UpdateAadAppDriver implements StepDriver {
         };
       }
       if (axios.isAxiosError(error)) {
+        if (
+          error.response!.status == 400 &&
+          error.response!.data.error.code === aadErrorCode.permissionErrorCode
+        ) {
+          const message = JSON.stringify(error.response!.data.error.message);
+          context.logProvider?.error(
+            getLocalizedString(logMessageKeys.failExecuteDriver, actionName, message)
+          );
+          return {
+            result: err(new HttpClientError(actionName, message, helpLink)),
```

Comment:
Shall we give out customized error message to ask user to update aad manifest?

Reply:
Like "Please confirm that the permission of the aad app in your project is consistent with the one already deployed on azure"

Reply:
Update Azure Active Directory app failed. Please double check the permission id you specified is correct or try again later. 

@MuyangAmigo